### DISCUSSION TITLE
Filesystem events won't let app load.

### DISCUSSION THREAD

**Discussion by vrishtrix** [coder/code-server#2272] [Q&A] [answered]
Recently I was trying to develop an android app using React Native with Expo. I got this problem where after bundling the app, when I opened it using the Expo mobile app, it just kept refreshing after the bundle was downloaded.

After some searching, I came across this post: [Expo: Infinite Refreshing without doing anything.](https://forums.expo.io/t/infinite-refreshing-without-doing-anything/2119)

_If there’s another program consistently making changes to files (like marking them as uploaded or another filesystem event), the packager may incorrectly pick that up as a change even if you aren’t editing the files actively._

So I was just wondering if this is the issue and code-server is  consistently making changes to files. I'm running the latest Docker image of code-server.

**EDIT:** Also, if this is the issue, do we have any workaround?

**Comment by code-asher** [comment, marked as answer]
I'm not sure if there's anything that could cause this on code-server's
end. Maybe it's the auto-save? Could try disabling that just to see if
it affects anything.

I tried to replicate with these steps but was unable:

1. Launch codercom/code-server:latest
2. Install yarn and node
3. yarn global add expo-cli
4. expo init my-project
5. select "tabs"
6. cd my-project
7. yarn start
8. Press w to open web
9. Wait for build then edit some files
10. It recompiles, but not infinitely for me

It could also be due to an extension so it might be worth trying without
any enabled if you haven't already.

**Comment by vrishtrix** [comment]
Hey there!

Thanks for the help. So I did figure it out a while before your post that it was the auto-save feature that was causing this. Also, I built for android and not web.

I really appreciate your help. Thank you very much! :D

**Reply by code-asher** [reply]
Awesome, glad to hear!
